client/src/pages/pot-instance/components/SimpleTimerView/FoodLegendList.tsx

```diff
@@ -0,0 +1,40 @@
+import DeleteIcon from '@mui/icons-material/Delete';
+import { IconButton, Stack } from '@mui/material';
+import timerStyles from 'pages/pot-instance/components/FoodTimerList/styles.module.css';
+import { ProgressData } from 'pages/pot-instance/models';
+import styles from './styles.module.css';
+
+interface Props {
+  colorMap: ProgressData[];
+  removeItem: (id: string) => void;
+}
+
+const LEGEND_ITEM_KEY = 'legendItem_';
+
+const FoodLegendList = ({ colorMap, removeItem }: Props) => {
+  return (
+    <Stack id={timerStyles.foodTimerContainer} spacing={1}>
+      {colorMap.map((legend) => (
+        <div
+          className={timerStyles.timerComponent}
+          key={LEGEND_ITEM_KEY + legend.id}
```

Comment:
Whats the purpose of the LEGEND_ITEM_KEY in this key?

Reply:
I wanted to align the prefix of the keys within the component. There are other/better ways of creating unique components keys so they don't clash with other components in the app.